components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -524,6 +526,19 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                                     {openLinkLabel}
                                 </button>
                             </div>
+                            {!useLatest && (
+                                <Alert type="info" className="mt-4 w-96">
```

Comment:
**thought**:  This is out of the scope of this PR but in the future we could probably introduce better breakpoints in general within the dashboard so that a) components like this expand with the container width and b) do not need to use a fixed width attribute.

However, that's minor for now and the effort/impact of doing this sounds low, but probably something to consider when we pick up https://github.com/gitpod-io/gitpod/issues/4050 for improving the responsive layout of the dashboard. Cc @jankeromnes as we've talked about this a couple of times.